include/unifex/connect_awaitable.hpp

```diff
@@ -297,6 +365,7 @@ struct _fn {
       (requires detail::_awaitable<Awaitable>)  //
       _sender<remove_cvref_t<Awaitable>>
       operator()(Awaitable&& awaitable) const {
+    // TODO: this is going to generate an unfortunate return address
```

Comment:
why's that?

Reply:
Because it's inside the bowels of Unifex here; I think a better return address might be `get_return_address(awaitable)`. We might have actually changed this in one of the branches that improves return address reporting for `task<>`.